components/composition/Grid/GridItem.ts

```diff
@@ -18,6 +18,7 @@ type Column =
 export interface GridItemAttributes {
   grow?: boolean;
   shrink?: boolean;
+  fill?: boolean;
```

Comment:
Could this be documented so it appears in the editor when the prop is used

```suggestion
  /**
   * Applies display grid so that children fill the height and width of the grid item
   */
  fill?: boolean;
```